CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,4 @@
+## Flux Community Code of Conduct## Flux Community Code of Conduct
```

Comment:
This should not repeat. And should be a H1 (even if if some other Flux repo have as a H2. Those should probably be changed as well).
```suggestion
# Flux Community Code of Conduct
```